riscv-asm.md

```diff
@@ -575,6 +580,8 @@ call offset                  | auipc x6, offset[31:12]; jalr x1, x6, offset[11:0
 tail offset                  | auipc x6, offset[31:12]; jalr x0, x6, offset[11:0]            | Tail call far-away subroutine
 fence                        | fence iorw, iorw                                              | Fence on all memory and I/O
 
+* [1] We don't specific code sequence for B-extension, since B-extension still not ratified or frozen, we'll specify the expansion sequence once it's frozen.
```

Comment:
```suggestion
* [1] We don't specify the code sequence when the B-extension is present, since B-extension still not ratified or frozen. We will specify the expansion sequence once it's frozen.
```